scene/main/node.cpp

```diff
@@ -1951,6 +1951,28 @@ Node *Node::get_child(int p_index, bool p_include_internal) const {
 	}
 }
 
+void Node::get_children_fast(LocalVector<Node *> &r_children, bool p_include_internal) const {
+	r_children.clear();
+	ERR_THREAD_GUARD
+	_update_children_cache();
+
+	if (p_include_internal) {
+		uint32_t num_children = data.children_cache.size();
+		r_children.resize(num_children);
+		for (uint32_t n = 0; n < num_children; n++) {
```

Comment:
Maybe worth tring a memcpy here, it would have a very good chance of using SIMD for the copy.